edn/src/types.rs

```diff
@@ -56,7 +56,6 @@ impl Display for Value {
             Integer(v) => write!(f, "{}", v),
             BigInteger(ref v) => write!(f, "{}N", v),
             Float(ref v) => {
-                // TODO: make sure float syntax is correct.
```

Comment:
Did you check that `format`'s rules for floats match EDN's? E.g., exponent capitalization and sign?